### ISSUE TITLE
Client not starting

### ISSUE THREAD

**Issue by robywoo** [debauchee/barrier#1866]
### What happened?

![image](https://user-images.githubusercontent.com/118034362/209961103-e812b127-c4ea-4ef3-a0fe-abba40ec588b.jpg)
SSL error when trying to connect to client. Failed to connect to secure socket. SSL certificate doesn’t exist. 
server is MACOS Ventura Client is Big Sur. Server runs but client doesn’t even if I manually enter the  IP in the client window. If I switch Big Sur as the server it runs but then same issue on Ventura as the client 
I don’t know enough to fix the SSL certificate 

### Version

v2.4.0

### Git commit hash (if applicable)

N/A

### If applicable, where did you install Barrier from?

GitHub.com/debauchee/barrier

### What OSes are you seeing the problem on? (Check all that apply)

macOS

### What OS versions are you using?

Ventura 13 and Big Sur 11.7

### Any other information

SSL certificate issue - I think

**Comment by Malachai7** [comment]
Since it's home network I turn off SSL. Have you tried turning that off feature?

**Comment by robywoo** [comment]
Where do I turn off SSL? Just in the Barrier settings? I ended up doing that but it eventually worked. Thanks for your advice

**Comment by nickmaglowsch** [comment]
how did you guys turned off ssl, the button is set as disabled in both server (windows) and client (macos) and i get the error on the client

**Comment by nickmaglowsch** [comment]
i just went back to 2.3.3 and it was all fine

**Comment by cleanunicorn** [comment]
SSL Fingerprint is already disabled.

**Comment by nickmaglowsch** [comment]
yeah on 2.4.0 in the UI says disabled but on logs show as enabled
